modal-js/test/image.test.ts

```diff
@@ -1,6 +1,15 @@
 import { App, Secret, Image } from "modal";
 import { expect, test } from "vitest";
 
+test("ImageFromId", async () => {
+  const app = await App.lookup("libmodal-test", { createIfMissing: true });
```

Comment:
nit: you don't need to create an app here I think! Also don't have to await `Image.fromId`.